### PR TITLE
Updating vm names

### PR DESCRIPTION
**Pull request by Shilpi-Das1**

### CODE REVIEW

File: vars/setupAndRunScaleTest.groovy [hunk 1]
```diff
@@ -33,10 +33,10 @@ def call(){
                 oc get pods -o wide -n openshift-image-registry ;
                 sleep 900 ;
                 oc get pods -o wide -n openshift-image-registry ;
-                echo './clusterbuster -d ${SCALE_NUM_OF_DEPLOYMENTS} -B ltccci ;while [ `oc get pods -n ltccci-0 | grep -v NAME | grep -v Running | wc -l` -ne 0 ];do sleep .0000000001;done' > deployments.sh;
+                echo './clusterbuster -d ${SCALE_NUM_OF_DEPLOYMENTS} -B ocp-cicd ;while [ `oc get pods -n ocp-cicd-0 | grep -v NAME | grep -v Running | wc -l` -ne 0 ];do sleep .0000000001;done' > deployments.sh;
```
Comment (sudeeshjohn) [inline vars/setupAndRunScaleTest.groovy:36]: this change is not necessary , its just the name of the pods

File: vars/setupAndRunScaleTest.groovy [hunk 2]
```diff
@@ -33,10 +33,10 @@ def call(){
...
                 chmod +x deployments.sh ;
                 /usr/bin/time -o ~/time_taken_deployments -f  'real-%E user-%U system-%S'  sh deployments.sh ;
-                ./clusterbuster --cleanup -B ltccci ;
+                ./clusterbuster --cleanup -B ocp-cicd ;
```
Comment (sudeeshjohn) [inline vars/setupAndRunScaleTest.groovy:39]: thi too

File: vars/setupAndRunScaleTest.groovy (hunk 1 above)
Comment (Shilpi-Das1) [reply vars/setupAndRunScaleTest.groovy]: @sudeeshjohn Addressed review comments

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Pull request merged**